jaeger-core/src/main/java/com/uber/jaeger/senders/UDPSender.java

```diff
@@ -85,14 +92,20 @@ int getSizeOfSerializedSpan(Span span) throws SenderException {
    */
   @Override
   public int append(Span span) throws SenderException {
-    int spanSize = getSizeOfSerializedSpan(span);
+    if (process == null) {
```

Comment:
`emitBatch()` is in `Sender.flush()`. Problem is that flush operates on thrift spans and I need access to tracer via Jaeger span.

Tracer tags should not change over time, therefore null check.